en_US/faq/faq.md

```diff
@@ -52,12 +52,12 @@ Besides using the configuration file (to configure authentication), EMQX can als
 
 
 
-## What's Hook? What's the use scenario?
+## What's a Hook and when is a Hook useful?
 
 **Tags:** [*WebHook*](tags.md#webhook)
 
 
-Hook is an interface provided by EMQX, which will be triggered when a connection, session or message is established/delivered. EMQX provides hooks listed in below, which allows user to save these triggered events to database, and user can conveniently query all kinds of information, such as client connect,  disconnect.  
+A Hook is an interface provided by EMQX, which will be triggered when a connection, session or message is established/delivered. EMQX provides the hook types listed below. Hooks allows a user to, for example, save events to database. This could be useful if one want to gather different kinds of information, such as when a client connect and disconnect.  
```

Comment:
```suggestion
A Hook is an interface provided by EMQX, which will be triggered when a connection, session or message is established/delivered. EMQX provides the hook types listed below. Hooks allows a user to, for example, save events to a database. This could be useful if one wants to gather different kinds of information, such as when a client connects and disconnects.  
```